science/bali-phy/Portfile

```diff
@@ -1,110 +1,39 @@
 PortSystem 1.0
-
+#PortGroup           github 1.0
 name                bali-phy
-version             2.1.1
-revision            5
-description         Bayesian Alignment and Phylogeny estimation
-long_description    BAli-Phy can estimate phylogenetic trees from sequence data when the alignment \
-                    is uncertain. Instead of conditioning on a single alignment estimate, BAli-Phy \
-                    accounts for alignment uncertainty by integrating over all alignments. \
-                    BAli-Phy does not rely on a guide tree because the alignment and the tree are \
-                    co-estimated. Therefore it can construct phylogeny estimates of widely \
-                    divergent sequences without bias toward a guide tree.
+version             3.0-beta6-20171223
```

Comment:
I just wanted to warn you that when version 3.0 gets released, you'll probably have to increase the epoch since version 3.0-beta6 is bigger than 3.0 in the eyes of MacPorts. No problem, just to let you know.

Reply:
Does using "~" instead of "-" solve this? e.g. 3.0~beta6-date?

Reply:
I'm afraid not, but I would be grateful if you brought this issue up again on the dev mailing list.

In your case you can arbitrarily invent the number since you're fetching from location where the version is not involved at all. In most cases the version number is part of the filename, so it cannot be arbitrarily changed. This is something where I would be grateful if we had a general mechanism to take care of prerelease versions. Often we put prerelease versions in `-devel` ports, but in this case of a completely broken old version, that makes no sense of course.

Again, this is by no mean an issue for you, an additional epoch field doesn't hurt anyone, more something that would be nice to support by MacPorts in general.